x-pack/plugins/cases/server/client/configure/get_fields.ts

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import Boom from '@hapi/boom';
-
-import { GetFieldsResponse } from '../../../common';
-import { ConfigureFields } from '../types';
-import { createDefaultMapping, formatFields } from './utils';
-
-export const getFields = async ({
-  actionsClient,
-  connectorType,
-  connectorId,
-}: ConfigureFields): Promise<GetFieldsResponse> => {
-  const results = await actionsClient.execute({
```

Comment:
Made this change to remove call to getFields, should've been part of this [bugfix PR](https://github.com/elastic/kibana/pull/97754) and was exposed as code rot when working on this PR

Reply:
@stephmilovic Don't we need any more the `fields`?

Reply:
@jonathan-buttner This change is gonna have conflicts with https://github.com/elastic/kibana/pull/95058

Reply:
Fixed!